src/web/reducers/flows.js

```diff
@@ -81,6 +84,18 @@ function createSnapshot(state) {
   }
 }
 
+function findNodesThatReferenceFlow(state, flowName) {
+  let nodes = []
+  for (let flow of _.values(state.flowsByName)) {
+    for (let node of flow.nodes) {
+      if (node.flow === flowName || _.find(node.next, { node: flowName })) {
+        nodes.push(node.id)
+      }
+    }
+  }
+  return nodes
+}
```

Comment:
Can also go fancy and rewrite this method as

```
const findNodesThatReferenceFlow = (state, flowName) =>
  _.flatten(
    _.values(state.flowsByName)
    .map(flow => flow.nodes)
  ).filter(
    node => node.flow === flowName || _.find(node.next, { node: flowName })
  ).map(node => node.id)
```

Reply:
Nice usage of flatten!